app/javascript/components/pages/YourTrips.js

```diff
@@ -11,28 +11,24 @@ export default class YourTrips extends Component {
         return trip.user_id === user.id;
       });
     return (
-      <div className="your-trips-container">
-        <h1 className="yt-header-title">Your Trips</h1>
-        <div className="yt-card">
+      <div className="main-card-container your-trips-container">
```

Comment:
```suggestion
      <div className="main-card-container">
```
 `your-trips-container` isn't being used